### DISCUSSION TITLE
does skywalking support topology group

### DISCUSSION THREAD

**Discussion by 2310652493** [apache/skywalking#11388] [Q&A] [answered]
![image](https://github.com/apache/skywalking/assets/88769942/20d58830-a6b1-4a54-bf3e-8a5a8f083508)
I want to group the programs in the topology diagram and how to do it

**Comment by wu-sheng** [comment, marked as answer]
No, we don't. We have limited the depth of the topology. If your topology is flat, there is no chance to set the boundary, because they are really connecting with each other directly.

**Reply by 2310652493** [reply]
Okay, thank you very much
